security/s2a/internal/conn/expander.go

```diff
@@ -0,0 +1,10 @@
+package conn
```

Comment:
Please put this file in `security/s2a/internal/crypter`. (All of the code for our record protocol will go in this directory, rather than a `conn` directory like ALTS has.)

Reply:
Done